lib/themes/dosomething/paraneue_dosomething/js/app.js

```diff
@@ -21,7 +21,6 @@ define("app", function (require) {
 
   require("campaign/Carousel");
```

Comment:
We might want to keep carousel for the Reportback Permalink and viewing multiple images on one permalink... cc @mikefantini